source/get-started/virtual-machine-install/hyper-v.rst

```diff
@@ -77,23 +83,22 @@ Create and configure new VM
    specifying the options below:
 
    - **Name**: Choose name (for example, ClearLinuxOS-VM)
-   - **Generation**: Generation 2
+   - **Specify Generation**: Generation 1
    - **Startup memory**: 2048 MB or more
+   - **Configure Networking**: Change :guilabel:`Connection` to `Default Switch`
    - **Connect Virtual Hard Disk**: Select :guilabel:`Use an existing virtual
      hard disk` and browse to find the :file:`clear-[VERSION]-hyperv.vhdx`
      file.
 
    After finishing the wizard, the VM will be created but not powered on.
 
 #. Configure the VM by right-clicking it in the Hyper-V Manager and selecting
-   :guilabel:`Settings...`.
-   Figure 3 shows the Settings page after configuration selections.
+   :guilabel:`Settings...`. Figure 3 shows the Settings page after configuration selections.
```

Comment:
Note that when using a Gen 1, as opposed to Gen 2 image, Hyper-V Manager does **not** present option for `firmware`, shown in previous version, Ln 92-93.